python-meh.spec

```diff
@@ -1,6 +1,3 @@
-%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
-%global with_python3 1
-
```

Comment:
You should replace `%define` below by `%global`. See [this](https://fedoraproject.org/wiki/Packaging:Guidelines#.25global_preferred_over_.25define) for more info.

Reply:
Done.